webui/react/src/types.ts

```diff
@@ -1,16 +1,18 @@
 import {
-  V1FittingPolicy, V1Pagination, V1ResourcePoolType, V1SchedulerType,
+  Determinedjobv1State, Determinedjobv1Type, V1FittingPolicy, V1Job, V1JobSummary,
+  V1Pagination, V1ResourcePoolType, V1RPQueueStat,
 } from 'services/api-ts-sdk';
+import * as gen from 'services/api-ts-sdk'; // DISCUSS wdy think about this instead?
```

Comment:
We have a few places that does this already but the line we use is something like this:
```
import * as Api from 'services/api-ts-sdk';
```

Reply:
if weback handles these well let's go with these instead. I'll got with Api